Firewood quest

NicodeH opened the pull request:
Issue : #6517 - Is firewood provided here?

Adding firewood quest

paulklie commented on app/src/androidMain/kotlin/de/westnordost/streetcomplete/quests/firewood/AddFirewood.kt:
```diff
@@ -0,0 +1,43 @@
+
+    override val elementFilter = """
+        nodes with
+        (leisure=firepit
```
could you improve the indentation here?

paulklie commented:
I think it might also be good to add a "It's private" option to the "Uh..." menu, to flag private fireplaces, or what do the others think?

NicodeH commented:
I've take into account your comments:

- Updated logo with multiple versions and shadow
- Added country filter
- Private option is not implemented yet — I'll let someone else handle this part

NicodeH commented:
The graphic was generated

paulklie commented:
It still includes a bunch of changes to unrelated to the changeset:

docs/
README 
build.gradle.kts 
values-fr/strings.xml

paulklie commented on app/src/androidMain/kotlin/de/westnordost/streetcomplete/quests/firewood/AddFirewood.kt:
```diff
@@ -0,0 +1,52 @@
+        "NO",
+        "CH"
+    )
+    override val changesetComment = "Specify yes or no the firewood is provided"
```
I think this could be shortened to "Specified if firewood is provided"

paulklie commented on app/src/androidMain/kotlin/de/westnordost/streetcomplete/quests/firewood/AddFirewood.kt:
```diff
@@ -0,0 +1,52 @@
+    override fun createForm() = YesNoQuestForm()
+
+    override fun applyAnswerTo(answer: Boolean, tags: Tags, geometry: ElementGeometry, timestampEdited: Long) {
+        tags["firepit:wood_provided"] = answer.toYesNo()
```
why "firepit:wood_provided" instead of just "wood_provided"?
It has 2 uses: https://taginfo.openstreetmap.org/keys/firepit%3Awood_provided
While wood_provided has 1k: https://taginfo.openstreetmap.org/keys/wood_provided

paulklie commented:
You can also hit "resolve" on any issue that you have fixed.

paulklie commented on app/src/androidMain/kotlin/de/westnordost/streetcomplete/quests/firewood/AddFirewood.kt:
```diff
@@ -0,0 +1,52 @@
+                or (amenity=bbq and fuel=wood)
+                or (tourism=wilderness_hut and fireplace=yes)
+            )
+            and access!=no or private
```
See here for the correct syntax: https://github.com/streetcomplete/StreetComplete/blob/aa9d7224c5d40028583217c2e64de196afb36324/app/src/androidMain/kotlin/de/westnordost/streetcomplete/quests/bicycle_repair_station/AddBicycleRepairStationServices.kt#L28

westnordost reviewed (commented):
Yeah, I do not like th icon much. But this is something I can fix myself when/after I merge this new quest.

Overall, good job, thank you for the contribution!
I  have but one minor thing to complain about, see below
